### PR TITLE
Ignore tx type 126 for celo

### PR DESCRIPTION
**Pull request by finleydecker**
Now that Celo is an OP chain, we need to ignore tx type 126 (0x7e).

The Celo chaintype is still useful because we still need to ignore tx types 123 (0x7b) and 124 (0x7c) on Celo specifically which we already have code for.

### CODE REVIEW

**Review by prashantkumar1982 — DISMISSED** [review]

File: core/chains/evm/gas/chain_specific.go
Comment (amit-momin) [inline core/chains/evm/gas/chain_specific.go:1]: Could you add this tx type to the block history tests [here](https://github.com/smartcontractkit/chainlink/blob/90f318bc52715909b9109f148947962a406f5314/core/chains/evm/gas/block_history_estimator_test.go#L1436-L1445)?

File: .changeset/few-sloths-explain.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+#updated ignoring txt type 126 on celo
```
Comment (amit-momin) [inline .changeset/few-sloths-explain.md]: nit: think this might be a type?
```suggestion
#updated ignoring tx type 126 on celo
```

**Review by amit-momin — DISMISSED** [review]

**Review by prashantkumar1982 — DISMISSED** [review]

**Review by george-dorin — DISMISSED** [review]

**Review by george-dorin — DISMISSED** [review]

File: core/chains/evm/gas/chain_specific.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ func chainSpecificIsUsable(tx evmtypes.Transaction, baseFee *assets.Wei, chainTy
 			return false
 		}
 	}
-	if chainType == chaintype.ChainOptimismBedrock || chainType == chaintype.ChainKroma || chainType == chaintype.ChainScroll || chainType == chaintype.ChainZircuit {
+	if chainType == chaintype.ChainOptimismBedrock || chainType == chaintype.ChainKroma || chainType == chaintype.ChainScroll || chainType == chaintype.ChainZircuit || chainType == chaintype.ChainCelo {
```
Comment (jmank88) [inline core/chains/evm/gas/chain_specific.go]: This is pretty long. What about a switch or slices.Contains?

File: core/chains/evm/gas/block_history_estimator_test.go [hunk 3]
```diff
@@ -1433,15 +1433,19 @@ func TestBlockHistoryEstimator_IsUsable(t *testing.T) {
 		assert.Equal(t, true, bhe.IsUsable(tx, block, defaultChainType, geCfg.PriceMin(), logger.Test(t)))
 	})
 
-	t.Run("returns false if transaction is of type 0x7c or 0x7b only on Celo", func(t *testing.T) {
+	t.Run("returns false if transaction is of type 0x7c, 0x7b, or 0x7e only on Celo", func(t *testing.T) {
 		tx := evmtypes.Transaction{Type: 0x7c, GasPrice: assets.NewWeiI(10), GasLimit: 42, Hash: utils.NewHash()}
-		assert.Equal(t, false, bhe.IsUsable(tx, block, chaintype.ChainCelo, geCfg.PriceMin(), logger.Test(t)))
+		assert.False(t, false, bhe.IsUsable(tx, block, chaintype.ChainCelo, geCfg.PriceMin(), logger.Test(t)))
```
Comment (amit-momin) [inline core/chains/evm/gas/block_history_estimator_test.go]: If we're switching to `assert.False` from `assert.Equal`, I think we'll need to remove this hard coded false. Same with the `assert.True`'s below
```suggestion
		assert.False(t, bhe.IsUsable(tx, block, chaintype.ChainCelo, geCfg.PriceMin(), logger.Test(t)))
```

**Review by george-dorin — APPROVED** [review]

**Review by simsonraj — APPROVED** [review]

**Review by amit-momin — APPROVED** [review]

**Pull request merged**